src/views/collections/CreateCollection/EditCollection.vue

```diff
@@ -0,0 +1,234 @@
+<template>
+  <EternaPage :title="$t('nav-bar:create-collection')">
+    <div class="page-content">
+      <div class="flex">
+        <div class="row">
+          <div class="col-md-6">
+            <CollectionInfo
+              :title="title"
+              :body="desc"
+              :picture="picture"
+              @update:title="(text) => (title = text)"
+              @update:body="(text) => (desc = text)"
+              @update:image="(file) => (newPicture = file)"
+            />
+          </div>
+          <div class="col-md-6">
+            <CollectionPuzzles
+              :puzzlelist="puzzlelist"
+              @update:puzzle="(puzzles) => (puzzlelist = puzzles)"
+            />
+          </div>
+        </div>
+        <div class="row">
+          <div class="col-md-6">
+            <b-button type="submit" variant="primary" @click="submit()">
+              {{ $t('update-collection:collection-info:main-action') }}
+            </b-button>
+            <b-modal id="delete" title="Delete Collection?" hide-footer>
+              <div class="d-block text-center">
+                Warning: This will permanently delete the collection!
```

Comment:
A good usability tip for making this more effective would be to require that they type in the name of their collection and have it match

Reply:
Should I do that check on the frontend or backend?

Reply:
Frontend is fine. This is to prevent user error, it's not for authorization